core/new-gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -109,6 +142,13 @@ export class ResultPanelComponent {
     modalComponentInstance.currentPageSize = this.currentPageSize;
   }
 
+  public getResult(): Object[] {
```

Comment:
remove these functions, they are only used in the HTML. just directly use the variables. try to reduce getter functions in the componenet class, if the variable is only used in HTML